spec/rails_cloud_tasks/rack/jobs_spec.rb

```diff
@@ -33,6 +33,15 @@
       its(:third)  { is_expected.to eq [{}.to_json] }
     end
 
+    context 'when the payload is null' do
+      let(:job_class) { 'DummyJobNoneArgs' }
+      let(:payload) { 'null' }
```

Comment:
This is actually not a "null payload". If you parse this, you'll get `nil`, which *may* be a valid, and even the *wanted*  argument used to start the job.

I think you would like to test a payload equal to `nil` or maybe `''`.
This is clearly going to make the parse call raise an exception.

In my opinion, it's easier to set a standard that all the jobs take a hash argument with whatever values they need to perform their tasks, if none is needed, they should receive an empty hash.

Reply:
I'm testing when the payload is `null`. It's different when the `payload` is missing. 

We can not guarantee "a standard behavior for Jobs", we should process either job with or without args.

I did not found another way to found out if the `perform` Job class_method  has no parameters.

Reply:
Then I suggest you rename the test case to `'when the payload is "null"'`.

But then, again, if I need my job to accept a `nil` argument, it won't work because this is removing the argument from the call

Reply:
I'm accepting other suggestions to do that.. do you have some?

Reply:
- Make all jobs take one argument of type hash. If none is needed, create the job with an empty hash (`payload: {}`)
- Make all jobs take a list of arguments. If none is needed, create the job with an empty list (`payload: []`). Start the execution with `job_class.perform_now(*args)`, `args` being the json-parsed list.

I personally prefer the first, because the arguments are named, which can facilitate should we debug the request or even check info on Cloud Tasks.

Reply:
again, WE CAN NOT force all jobs to have at least one job. It's a client implementation

Reply:
we can rescue the `ArgumentError` and call `job.perform_now`. What do you think?